### ISSUE TITLE
5.1 does not open scenes

### ISSUE THREAD

**Issue by sunnydrake** [superpowers/superpowers-app#63]
Tab opens and im stuck with LOADING black screen.
scene consist of 3d models with shaders,lights and cam created in 4.0.
new scene created in 5.1 also stuck
build is superpowers-v5.1.0-linux-x64.zip version file v1.7.9

**Comment by bilou84** [comment]
Is that only an issue with scenes? I can't seem to reproduce that on Windows, so could be a Linux only issue.
You can press F12 to open some dev tools, and then in the console tab there could be some errors logged here that might be useful.

Also, in the mean time you can use the previous version 4.0. The new version only add a new piece of API that isn't used by the game system yet.

**Comment by sunnydrake** [comment]
here is error from F12 console

SupEngine.js:43427 Uncaught TypeError: Cannot read property 'domElement' of undefined
    at Camera.computeAspectRatio (SupEngine.js:43427)
    at new Camera (SupEngine.js:43438)
    at Object.7../network (index.js:3264)
    at s (index.js:1)
    at index.js:1
    at Object.10../engine (index.js:3733)
    at s (index.js:1)
    at index.js:1
    at Object.9.../../components/SceneUpdater (index.js:3492)
    at s (index.js:1)

btw is there any way to manipulate vertex of 3d model?:) sorry for out of topic

**Comment by sunnydrake** [comment]
to my surprise going with chrome last to http://127.0.0.1:4237 actually works.. so it may do something with build in chrome

**Comment by sunnydrake** [comment]
running an app says my device does  not support webGL renderer .. definetly some flag compile chrom issue

**Comment by bilou84** [comment]
Yeah I updated the version of [electron](https://github.com/electron/electron) we use, which is basically the wrapper we use for chrome. This specific version probably trigger the bug on your end for some reasons.
One solution would be to find out since which version of electron it breaks. But I can't do that myself since I can't reproduce the bug. If you have the time and have the tinkering mind, you can try and compile this app yourself and use an older version of electron (we use 1.7.9) to try and get one working.

> btw is there any way to manipulate vertex of 3d model?:) sorry for out of topic

And for this question, sadly not easily. Our 3d support is quite limited.

**Comment by sunnydrake** [comment]
unfortunetly changing package.json line to "types/electron": "1.4.38" and 1.6.10,1.3.22 does not fix problem 
build-in version is 1.6.10 last version is 1.8.2-beta + official 1.7.x branch(dunno how to install ^1.7.9)
btw i can provide teamviewer or remotedesktop user account if you like.

**Comment by sunnydrake** [comment]
updated to superpowers 5.2 same issue exist.
